spec/models/sponsor_spec.rb

```diff
@@ -328,6 +328,24 @@
       expect(request_fulfilled_sponsor.eligible_for_sponsorship?).to eq false
     end
 
+    describe '#sponsorship_changed!', :mytest => true do
```

Comment:
Since you always run `$ rspec` before you are ready to `$ git push` you will immediately notice if only a few tests are running.